ow_faults_detection/src/FaultDetector.cpp

```diff
@@ -261,7 +261,7 @@ void FaultDetector::cameraRawCb(const sensor_msgs::Image& msg)
 void FaultDetector::powerTemperatureListener(const BatteryTemperature& msg)
 {
   // check for excessive battery temperature
-  if (msg.value > POWER_THERMAL_MAX) {
+  if (msg.value >= POWER_THERMAL_MAX) {
```

Comment:
Minor point, but wasn't the original expression correct?

Also assuming this change is unrelated to the feature at hand.

Reply:
Never mind, I looked at more code and get it now :)

Reply:
Ah, I forgot to mention that in the summary. Ultimately I think it's down to a basic interpretation of the thermal failure threshold: do we want it to trigger when the temperature hits 70C, or do we want it to trigger when the temperature exceeds 70C?

I can change it back if the latter is preferred, but the other power faults use the former's logic (low state of charge triggers when the SoC hits 10%, not when it goes under 10%, & ICL triggers when the total loss in one cycle hits 5% or higher, not when it exceeds 5%), so it's somewhat of a consistency change.